tests/tck/features/bugfix/VariableExpression.feature

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
```

Comment:
Remove the CC license

Reply:
ACK.

Reply:
It seems you haven't fix the license declaration.